### ISSUE TITLE
"Settings" in the sidebar is odd

### ISSUE THREAD

**Issue by madsrh** [Feichtmeier/settings#136]
I've mentioned this is https://github.com/Feichtmeier/settings/issues/28 but we never got around to discussing it. IMHO the "Search" label above the sidebar is unnecessary. I would be better to just add a text to show the user that there's a textfield.

![search](https://user-images.githubusercontent.com/3986894/136108889-45e9fce7-aaff-459a-b6be-5c62bac20ec6.gif)

**Comment by Feichtmeier** [comment]
True! As long as we keep that gtk3 headerbar at the top, the double "settings" is stupid

Maybe maybe, some far time in the future when we have enough flutter apps in ubuntu we can overwrite the gtk3 headerbar with a flutter widget and do our own stuff in this region. But this is hypothetical

**Comment by Feichtmeier** [comment]
@madsrh but if the search field is already visible  - what should the button with the 🔍  actually reveal then?

**Comment by Feichtmeier** [comment]
@madsrh 

What about this?

![grafik](https://user-images.githubusercontent.com/15329494/136545026-b4ec4f47-19f1-4767-8f65-cf709b16b3c5.png)

-->

![grafik](https://user-images.githubusercontent.com/15329494/136545123-ec4abfe1-2f99-4e6c-8c2f-760299225479.png)
